Apps/Examples/Examples/All Examples/RasterTileSourceExample.swift

```diff
@@ -34,6 +34,7 @@ class RasterTileSourceExample: UIViewController, ExampleProtocol {
 
         // Specify the tile size for the `RasterSource`.
         rasterSource.tileSize = 256
+//        rasterSource.tileRequestsDelay = 5000
```

Comment:
This doesn't work, because the tileRequestsDelay property is a runtime property and must be set after the source is added to the style.

In Android, in each generated Source class, we save two list of source properties: one for normal `sourceProperties`, which are added while calling `style.addSource`; and one for `volatileSourceProperties`, which are added after the source is added to the style using `style.setSourceProperties`.

So I think the generated properties like `prefetchZoomDelta`, `minimumTileUpdateInterval`, `maxOverscaleFactorForParentTiles` will not work on iOS, since all these source properties are treated as normal source properties and applied while calling `style.addSource`.

cc @macdrevx